### DISCUSSION TITLE
插件页无法使用，稍后读页面图标全部白色看不清或兑换页无输入框的解决方案

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2906] [重要通知]
## 问题现象

> 打开稍后读如下所示

![060c6b523977384a8fa9311aedc0069](https://user-images.githubusercontent.com/30795466/133557243-41ca5f90-0961-4b44-aba6-9e6d72627405.png)

> 在插件中心https://simpread.ksria.cn/plugins/  点插件下载毫无反应，如下图

![a55167364ee1a9eb65e4e926e71a90d](https://user-images.githubusercontent.com/30795466/133557354-fcb13cab-908e-4360-9502-fc4986c738ef.png)

> 登录 https://simpread.pro/code 无登录框

![image](https://user-images.githubusercontent.com/81074/135441497-3089a1be-66b3-477d-96f3-117d4f040e97.png)

## 问题原因

造成这个情况的是无法访问 `cdnjs.cloudflare.com` （遇到此问题时，请通过 [此链接](http://cdnjs.cloudflare.com/) 自查）

## 解决方案（首选）

目前简悦相关网站已经修改到国内线路，包括

1. [x] https://simpread.pro 相关页面均已经恢复
2. [x] https://simpread.ksria.cn/plugins https://simpread.ksria.cn/sites 相关页面均已经恢复
3. [ ] 扩展端和同步助手暂时没有解决办法，请确保可以访问  `cdnjs.cloudflare.com`

## 关联问题

- https://github.com/Kenshin/simpread/issues/3616
- https://github.com/Kenshin/simpread/issues/3619
- https://github.com/Kenshin/simpread/issues/1415

**Comment by freshcn** [comment]
我发现我就算是修改为114.114.114.114 这个域名也是被解析到了127.0.0.1

**Reply by freshcn** [reply]
![image](https://user-images.githubusercontent.com/1764041/140922286-493c7017-347c-4df9-911a-2e1bda33e4d8.png)

**Reply by binsee** [reply]
你可以试试腾讯 `119.29.29.29 `或阿里 `223.5.5.5`

**Reply by freshcn** [reply]
目前看来只有阿里的DNS还是正确的。其他家的DNS都是127.0.0.1

**Reply by binsee** [reply]
> 目前看来只有阿里的DNS还是正确的。其他家的DNS都是127.0.0.1

至少我这边腾讯和阿里的都正常，返回的是一样的 `ipv4 + ipv6`。如果你那边返回不一致，或许是存在dns劫持

**Reply by DruculaC** [reply]
我提供一个信息，使用家庭网络可以看到输入框，但是在公司打开，用的是电信专线（公司网络）就看不到输入框了。然后测试使用电信5G网络，也是可以看到输入框的。使用公司网络固定IP时，无论是edge或者firefox浏览器都看不到输入框。

**Reply by Kenshin** [reply]
@DruculaC 

哈哈，谢谢提供进一步思路。

兑换平台的话，只要是能访问 https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js 就能看到输入框

**Reply by freshcn** [reply]
对应的这些资源是否可以使用国内的CDN资源呢？

**Reply by Kenshin** [reply]
@freshcn 

会考虑的。

**Reply by gujin03** [reply]
我和上面@DruculaC这位情况一样，在公司用的专线出现的此种情况。

**Reply by Kenshin** [reply]
> 我和上面@DruculaC这位情况一样，在公司用的专线出现的此种情况。

嗨，同一个问题不要在多个地方同时提问... 这是你要的答案 https://github.com/Kenshin/simpread/issues/3597#issuecomment-1078863044
